src/transformers/generation/utils.py

```diff
@@ -567,24 +570,6 @@ def _prepare_decoder_input_ids_for_generation(
 
         return decoder_input_ids, model_kwargs
 
-    def _get_decoder_start_token_id(
```

Comment:
The logic of this function is now within `_prepare_special_tokens`, which preprocesses all special tokens